packages/ui/src/Alert.module.scss

```diff
@@ -0,0 +1,153 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+// TODO: potential max-height
+.alert {
+  display: inline-block;
+  border-radius: c.$borderRadius;
+  padding: c.$grid * 2.5;
+  width: c.$alertWidth;
+
+  .header {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    .icon {
+      margin-right: c.$grid * 2.5;
+    }
+
+    .title {
+      margin-right: c.$grid * 8.5;
+
+      @include c.typographyHighlightFont;
+      @include c.typographyBodyNormal;
+      @include h.overflowEllipsis;
+    }
+
+    .close {
```

Comment:
The parent container here is flex. Why do we need to make this element absolute? Wouldn't flex parent position it correctly if we added `flex: 1 0 auto;` to `title`?

Reply:
Good thinking!